library/fedora

```diff
@@ -22,27 +22,27 @@ arm32v7-Directory: armhfp/
 arm64v8-Directory: aarch64/
 ppc64le-Directory: ppc64le/
 
-Tags: rawhide
-Architectures: amd64, arm64v8, ppc64le
-GitFetch: refs/heads/rawhide
-GitCommit: 3638c329a8590f71b2922b24494f66b6c932d8d2
+Tags: rawhide, 30
+Architectures: amd64, arm64v8
+GitFetch: refs/heads/30
+GitCommit: f65a23bf343b7bb6b1e32e46ba8e1157f23761fa
 amd64-Directory: x86_64/
 arm64v8-Directory: aarch64/
-ppc64le-Directory: ppc64le/
 
-Tags: latest, 28
-Architectures: amd64, arm32v7, arm64v8, ppc64le
```

Comment:
@cverna Any reason for removing `arm32v7` support for Fedora 28+?

Reply:
The build is currently failing, this will be back once we manage to fix it